golem_messages/message/concents.py

```diff
@@ -147,6 +147,67 @@ class FileInfo(datastructures.FrozenDict):
         }
 
 
+class ForceGetTaskResult(base.Message):
+    TYPE = CONCENT_MSG_BASE + 6
+
+    __slots__ = [
+        'report_computed_task',
```

Comment:
According to the documentation (Dokumentacja analityczna integracji Golem-Concent) a field is missing: there should be also an optional MessageForceReportComputedTask.